components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/internal/OrganizationManagementHandlerServiceComponent.java

```diff
@@ -0,0 +1,118 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.organization.management.handler.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.wso2.carbon.identity.event.handler.AbstractEventHandler;
+import org.wso2.carbon.identity.event.services.IdentityEventService;
+import org.wso2.carbon.identity.governance.IdentityGovernanceService;
+import org.wso2.carbon.identity.organization.management.handler.OrganizationCreationHandler;
+import org.wso2.carbon.identity.organization.management.service.OrganizationManager;
+
+/**
+ * Organization management handler data holder.
+ */
+@Component(
+        name = "identity.organization.management.handler.component",
+        immediate = true
+)
+public class OrganizationManagementHandlerServiceComponent {
+
+    private static final Log log = LogFactory.getLog(OrganizationManagementHandlerServiceComponent.class);
+
+    /**
+     * Register the organization mgt listener implementation in the OSGI context.
+     *
+     * @param componentContext OSGi service component context.
+     */
+    @Activate
+    protected void activate(ComponentContext componentContext) {
+
+        try {
+            BundleContext bundleContext = componentContext.getBundleContext();
+            bundleContext.registerService(AbstractEventHandler.class.getName(), new OrganizationCreationHandler(),
+                    null);
+
+            if (log.isDebugEnabled()) {
+                log.debug("Organization management ext component activated successfully.");
+            }
+        } catch (Throwable e) {
+            log.error("Error while activating organization management ext module.", e);
+        }
+    }
+
+    @Reference(
+            name = "identity.event.service",
+            service = IdentityEventService.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetIdentityEventService"
+    )
+    protected void setIdentityEventService(IdentityEventService identityEventService) {
+
+        OrganizationManagementHandlerDataHolder.getInstance().setIdentityEventService(identityEventService);
+        if (log.isDebugEnabled()) {
+            log.debug("IdentityEventService set in Central logger.");
```

Comment:
```suggestion
            log.debug("IdentityEventService set in organization management handler .");
```
change according to the service name

Reply:
Resolved in [8672e33](https://github.com/wso2-extensions/identity-organization-management/pull/205/commits/8672e33eaea07f78bf4e119a97f86b1268301753)